src/patternfly/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -269,6 +269,23 @@ import './MenuToggle.css'
 {{/menu-toggle}}
 ```
 
+### Small
+```hbs
+{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--IsSmall=true menu-toggle--attribute='aria-label="Actions"' menu-toggle--icon="ellipsis-v"}}
```

Comment:
for `menu-toggle--icon`, since we're hardcoding an svg then `menu-toggle--HasIcon` or something might be more fitting. Here it makes it look like we can pass in a string to determine what icon gets rendered.